### DISCUSSION TITLE
fetchExternalImage timesout and no way to change it

### DISCUSSION THREAD

**Discussion by nrandell** [vercel/next.js#72072] [Help]
### Summary

I've just upgraded to next 15 and am having issues in local dev where I'm getting a lot of 500s for various images. I've narrowed it down to fetchExternalImage timing out after 7 seconds.

If I hand modify image-optimizer.js and change the timeout to 70 seconds, all works ok. 

Is there any way to override this timeout?

**Comment by reiv** [comment]
+1. It looks like the timeout was changed from unbounded to 7 seconds in https://github.com/vercel/next.js/pull/65821. Some options were later added in https://github.com/vercel/next.js/commit/0dce6bb4730de7544742a6d8806fd23f45160a9e, but those are only concerned with Sharp, not the upstream fetch.

IMO it would make sense to change the hardcoded value to also use  `experimental.imgOptTimeoutInSeconds`.

We are forced to work with an unreliable image provider which frequently incurs response times over 7 seconds.

**Reply by nrandell** [reply]
How can we raise this as an issue? Issues need to be reproducible, but this is a transient error, so no reproducible link!
